upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -226,8 +226,13 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 
 	if b.cluster.Spec.Networking.Flannel != nil {
 		key := "networking.flannel"
-		version := "0.7.0"
-
+		var version string
+		switch {
+		case kv.Major == 1 && kv.Minor <= 5:
+			version = "0.7.0"
+		default:
+			version = "0.7.0_1.6"
```

Comment:
I'm trying to figure out how to deal with this... I think we need to reinstall whenever the k8s version changes, which will require some changes to the infrastructure.  Sorry for the delay.